### ISSUE TITLE
"No complete Backup" since Veeam v9.5.0.1536

### ISSUE THREAD

**Issue by marvinwankersteen** [davidedg/check_mk_plugins#2]
Since we upgraded to Veeam B&R v9.5.0.1536, every "`VEEAM Client ...`"-service reports "`No complete Backup`".

Here are an output from "`cmk -d vcenter`"
```
<<<<servername>>>>
<<<veeam_client:sep(9)>>>
Status	Pending
JobName	Linux_VMs_(Copy)
TotalSizeByte	0
ReadSizeByte	0
TransferedSizeByte	0
StartTime	
StopTime	
DurationDDHHMMSS	00:12:05:28
AvgSpeedBps	0
DisplayName
```

In CheckMK it looks like this:
![screenshot_-_area_-_2018-01-04_-_10-16-32](https://user-images.githubusercontent.com/17569312/34556900-8e2f5fb6-f138-11e7-9352-3e559c84a4a0.png)

If you need more output, tell me.

**Comment by davidedg** [comment]
Hi
I am aware of the problem, another user already reported it.
Unfortunately I cannot look into it soon, maybe in february.
Can you open issue on github?

Bye

Il 04 gen 2018 10:19 AM, "marvinwankersteen" <notifications@github.com> ha
scritto:

**Comment by davidedg** [comment]
I found the root cause:

In v9.5U3 they changed the "StartTime" and "StopTime" attributes.
There's a reference of this change also in [this post](https://forums.veeam.com/powershell-f26/getting-backup-size-per-vm-t23342.html#p265778) 

As a quick and dirty workaround, you can replace all occurences of
`Progress.StartTime` with `Progress.StartTimeLocal`
and
`Progress.StopTime` with `Progress.StopTimeLocal`

I will need to insert a switch/version check to support both versions of course.

I also noticed that the attribute `DisplayName` has been removed.
This does not seem to be important, because was not actually implemented on the server-side check (I only developed this client-side check, not the server counterpart).

**Comment by davidedg** [comment]
should be fixed with new revision 1.4
